### PR TITLE
Create Apache Beam and Model Handlers compatibility table

### PR DESCRIPTION
**Pull request by AnandInguva**
TODO: fill out the versions.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by tvalentyn** [comment]
I haven't paid attention to the version we support, would defer to @damccorm. 
nit: please  fix whitespace failures.

**Comment by AnandInguva** [comment]
Converted to draft. Will pick it up once the 2.47.0 cut is done

File: website/www/site/content/en/documentation/sdks/python-machine-learning.md [hunk 1]
```diff
@@ -262,6 +262,24 @@ Use elements of the same size or resize the inputs. For computer vision applicat
...
+## Compatible versions
+
+The following table describes how the `RunInference` Model Handlers versions are compatible with
+its dependency PyPI packages. This is determined by our testing framework,
+but other *untested* combinations may also work.
+
+| apache_beam                                                            | Tensorflow    | TFX-BSL            | PyTorch  | Scikit-learn | TensorRT             | Onnx   |
+|------------------------------------------------------------------------|---------------|--------------------|----------|--------------|----------------------|--------|
+| [GitHub master](https://github.com/apache/beam/blob/master/RELEASE.md) | \>=2.12.0     | WIP                | \>=1.9.0 | \>=1.0.0     | \>=8.5.1.7,<=8.5.3.1 | WIP    |
+| [2.46.0](https://github.com/apache/beam/tree/v2.46.0)                  | \>=2.9,<=2.11 | \>=1.10.0,<=1.12.0 | \>=1.9.0 | \>=1.0.0     | \>=8.5.1.7,<=8.5.3.1 | 1.13.0 |
```
Comment (tvalentyn) [inline website/www/site/content/en/documentation/sdks/python-machine-learning.md:274]: add 2.47.0 ?

File: website/www/site/content/en/documentation/sdks/python-machine-learning.md [hunk 2]
```diff
@@ -262,6 +262,24 @@ Use elements of the same size or resize the inputs. For computer vision applicat
 Disable batching by overriding the `batch_elements_kwargs` function in your ModelHandler and setting the maximum batch size (`max_batch_size`) to one: `max_batch_size=1`. For more information, see
 [BatchElements PTransforms](/documentation/sdks/python-machine-learning/#batchelements-ptransform). For an example, see our [language modeling example](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/inference/pytorch_language_modeling.py).
 
+## Compatible versions
+
+The following table describes how the `RunInference` Model Handlers versions are compatible with
```
Comment (tvalentyn) [inline website/www/site/content/en/documentation/sdks/python-machine-learning.md:267]: Wording suggestion: The following table describes the tested configurations of the `RunInference` Model Handlers across different versions of Beam and dependent libraries.  The tested configurations are considered compatible, but other (untested) combinations may also work.

**Pull request closed without merging**